plugins/entity-feedback/src/components/FeedbackResponseDialog/FeedbackResponseDialog.tsx

```diff
@@ -105,44 +127,73 @@ export const FeedbackResponseDialog = (props: FeedbackResponseDialogProps) => {
+        <FormControl component="fieldset" fullWidth>
+          <FormLabel component="legend">Select all that apply</FormLabel>
+          <FormGroup className={classes.boxContainer}>
+            {feedbackDialogResponses.map((response: EntityFeedbackResponse) => (
+              <Grid container key={response.id} direction="column" spacing={1}>
+                <FormControlLabel
+                  control={
+                    <Checkbox
+                      checked={responseSelections[response.id]}
+                      disabled={saving}
+                      name={response.id}
+                      onChange={e =>
+                        setResponseSelections({
+                          ...responseSelections,
+                          [e.target.name]: e.target.checked,
+                        })
+                      }
+                      color="primary"
+                    />
+                  }
+                  label={response.label}
+                />
+                <Collapse in={responseSelections[response.id]}>
+                  <TextField
+                    data-testid={`feedback-response-dialog-collapse-comments-input-${
+                      responseSelections[response.id]
+                    }`}
                     disabled={saving}
-                    name={response.id}
+                    className={classes.commentBoxes}
+                    multiline
+                    minRows={2}
+                    fullWidth
+                    variant="outlined"
+                    value={comments[response.id] || ''}
                     onChange={e =>
-                      setResponseSelections({
-                        ...responseSelections,
-                        [e.target.name]: e.target.checked,
-                      })
+                      setComments(prevComments => ({
+                        ...prevComments,
+                        [response.id]: e.target.value,
+                      }))
                     }
                   />
-                }
-                label={response.label}
-              />
+                </Collapse>
+              </Grid>
             ))}
           </FormGroup>
         </FormControl>
         <FormControl fullWidth>
+          <FormLabel component="legend" className={classes.formLabel}>
+            Additional comments
+          </FormLabel>
           <TextField
             data-testid="feedback-response-dialog-comments-input"
             disabled={saving}
-            label="Additional comments"
             multiline
             minRows={2}
-            onChange={e => setComments(e.target.value)}
+            onChange={e =>
+              setComments(prevComments => ({
+                ...prevComments,
+                ['additional']: e.target.value,
+              }))
+            }
             variant="outlined"
-            value={comments}
+            value={comments.additional || ''}
```

Comment:
I a bit concerned about hard coding the key `additional` here as there is a possibility that this may clash with user supplied custom `EntityFeedbackResponse` that also uses the same `additional` id 

Perhaps we should separate the custom response comments like so:
```ts
export interface Comments {
  responseComments: {
    [key: string]: string;
  };
  additionalComments?: string;
}
```